dolphin/db/sqlalchemy/api.py

```diff
@@ -173,7 +173,17 @@ def access_info_create(context, values):
 
 def access_info_update(context, access_info_id, values):
     """Update a storage access information with the values dictionary."""
-    return NotImplemented
+    session = get_session()
+    try:
+        with session.begin():
+            result = session.query(AccessInfo) \
+                .filter(AccessInfo.storage_id == access_info_id) \
+                .first().update(values)
+    except Exception as e:
+        session.rollback()
```

Comment:
I think session.rollback() is not required, because with session.begin() will help you do it. try catch may not be required

Reply:
yes done, checked it by forcing excepion form with block.. session management will take care of rollback()..